sdk/eidolon_ai_sdk/builtins/code_builtins.py

```diff
@@ -68,7 +69,6 @@ def named_builtins():
     builtin_list = [
         AgentMachine,
         # security manager
-        (SecurityManager, SecurityManager),
```

Comment:
we don't need to make a mapping of securitymanager -> security manager, so I pulled this out. The single security manager below is all that is needed since there is no base class.